docker/src/main/java/org/jclouds/docker/compute/functions/ImageToImage.java

```diff
@@ -91,9 +91,15 @@ public OsFamily apply(final String description) {
       };
    }
 
-   private String parseVersion(String description) {
-      String version = get(Splitter.on(":").split(description), 1);
-      logger.debug("os version for item: %s is %s", description, version);
+   private String parseVersion(String repoTag) {
+      String version;
+      if (repoTag.contains(":")) {
+         version = get(Splitter.on(":").split(repoTag), 1);
+      } else {
+         version = "<none>";
```

Comment:
I will get rid of the whole `parseVersion` method. It's still not correct. There can be `repoTag` with port provided for repo-host (e.g. `internal-repository.company.example:8888/a/b/c/d:test`)